object/object.go

```diff
@@ -112,9 +120,9 @@ func (f *Function) Inspect() string {
 	out.WriteString("f")
 	out.WriteString("(")
 	out.WriteString(strings.Join(params, ", "))
-	out.WriteString(") {\n")
+	out.WriteString(") {")
```

Comment:
why removing the new lines?

Reply:
This is related to the `error == expected` testing problem.  `object.Function.Inspect()` was returning a result containing gratuitous newlines that complicate matching the expected result in the tests. I don't understand why the internal representation of a function should contain newlines (newlines are for people, not for parsers :-)). I also don't see that there is any other user of `Inspect()` other than the testing module and so `Function.Inspect()` seems to be the only one doing this kind of formatted result string.